src/Engines/QueryBuilderEngine.php

```diff
@@ -103,7 +103,7 @@ public function count()
 
         // check for select soft deleted records
         if (! $this->withTrashed && $this->modelUseSoftDeletes()) {
-            $myQuery->whereNull($myQuery->getModel()->getTable() . '.deleted_at');
+            $myQuery->whereNull($myQuery->getModel()->getQualifiedDeletedAtColumn());
```

Comment:
We might need to still include the table name to avoid ambiguous column issues:

```php
$myQuery->whereNull( $myQuery->getModel()->getTable() . '.' . $myQuery->getModel()->getQualifiedDeletedAtColumn());

```

Thanks!